src/plugins/dashboard/public/dashboard_actions/index.ts

```diff
@@ -24,6 +24,8 @@ interface BuildAllDashboardActionsProps {
   plugins: DashboardStartDependencies;
 }
 
+export const DASHBOARD_ACTION_GROUP = { id: 'dashboard_actions', order: 10 };
```

Comment:
**nit:** Since this is a constant object, we should make it [deeply immutable](https://learntypescript.dev/10/l5-deep-immutable) - otherwise, we can do stuff like `dashboardActionGroup.id = 'somethingNew'` which is not ideal.

```suggestion
export const DASHBOARD_ACTION_GROUP = { id: 'dashboard_actions', order: 10 } as const;
```